db/primary_migrate/20220705152334_add_address_and_location_to_in_person_enrollment.rb

```diff
@@ -0,0 +1,6 @@
+class AddAddressAndLocationToInPersonEnrollment < ActiveRecord::Migration[6.1]
+  def change
+    add_column :in_person_enrollments, :current_address_matches_id, :boolean, comment: "True if the user indicates that their current address matches the address on the ID they're bringing to the Post Office."
+    add_column :in_person_enrollments, :selected_location_details, :jsonb, comment: "The location details of the Post Office the user selected (including title, address, hours of operation)"
```

Comment:
Does USPS give us any location identifier we could use instead of storing a blob of the full details? A concern I'd have is that we'd be storing a lot of duplicate data with this.

Reply:
Unforunately there's no API to look up locations by ID at this point

Reply:
No, we won't need to do a look-up with this field data, for now.